client/template.js

```diff
@@ -1,4 +1,6 @@
 module.exports = async(name, title = '', props = {})=>{
+	const HOMEBREWERY_PUBLIC_URL=props.publicUrl ?? '';
```

Comment:
With the change below, this can be changed once more:

```suggestion
	const HOMEBREWERY_PUBLIC_URL=props.publicUrl;
```

Reply:
Implemented in the latest commit.